controllers/admin/AdminCmsCategoriesController.php

```diff
@@ -221,11 +221,11 @@ public function renderForm()
                     'name' => 'name',
                     'required' => true,
                     'lang' => true,
-                    'hint' => $this->l('Invalid characters:').' &lt;&gt;;=#{}'
+                    'hint' => $this->trans('Invalid characters:', array(), 'Admin.Notifications.Error').' &lt;&gt;;=#{}'
```

Comment:
Are you sure?

Usually these are what the levels mean:
- info, verbose stuff, can be ignored
- warning, can be ignored for now but you should do something
- error, can't go on and you must do something

PrestaShop isn't able to process the form, so looks like an error to me.

Do you agree this should be an error notification after this explanation? ^^